website/docs/cloud-docs/integrations/kubernetes/features.mdx

```diff
@@ -0,0 +1,150 @@
+---
+page_title: Terraform Cloud Operator for Kubernetes features
+description: >-
+  Terraform Cloud Operator allows you to provision infrastructure directly from
+  the Kubernetes control plane.
+---
+
+# Supported Terraform Cloud Features
+
+The Terraform Cloud Operator for Kubernetes allows you to create agent pools, deploy modules, and manage workspaces through Kubernetes controllers. These controllers enable you to automate and manage Terraform Cloud resources using custom resources in Kubernetes.
+
+## Agent Pool
+
+Agent pools in Terraform Cloud manage the execution environment for Terraform runs. The Terraform Cloud Operator allows you to create and manage agent pools as part of your Kubernetes infrastructure.
+
+The following example creates a new agent pool with the name `agent-pool-development` and generates an agent token with the name `token-red`.
+
+```yaml
+---
+apiVersion: app.terraform.io/v1alpha2
+kind: AgentPool
+metadata:
+  name: this
+spec:
+  organization: kubernetes-operator
+  token:
+    secretKeyRef:
+      name: tfc-operator
+      key: token
+  name: agent-pool-development
+  agentTokens:
+    - name: token-red
+```
+
+The operator stores `token-red` agent token in a Kubernetes secret named TODO.
```

Comment:
@arybolovlev I'm having a hard time remembering what the secret will be named. Do you know what the secret name would be in this example? Thanks!

Reply:
The secret name uses the following template: `<metadata.name>-agent-pool`, in this specific case it will be `this-agent-pool`.